### PR TITLE
[lis-rpm] Add local hyper-v perf trigger

### PR DESCRIPTION
**Pull request by mihaico**

### CODE REVIEW

File: scripts/lis_hyperv_platform/trigger_lis_perf_tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+param(
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/trigger_lis_perf_tests.ps1]: can you reuse \scripts\\lis_hyperv_platform\\trigger_perf_tests.ps1" for this purpose?

Comment (mihaico) [reply scripts/lis_hyperv_platform/trigger_lis_perf_tests.ps1]: fixed

File: linux_pipeline/Jenkinsfile_lis_rpm_hyperv [hunk 2]
```diff
@@ -119,6 +122,31 @@ stage ("Deploy stage") {
     }
 }
 
+stage('Local Hyper-V performance') {
```
Comment (mbivolan) [inline linux_pipeline/Jenkinsfile_lis_rpm_hyperv]: declarative pipeline sintax won't worn in this case.
you can use somthing like: 
stage (){
    node(){
         withcreds (){
              powershell
         }
    }
}

File: linux_pipeline/Jenkinsfile_lis_rpm_hyperv [hunk 3]
```diff
@@ -15,7 +15,10 @@ properties ([
         [$class: 'StringParameterDefinition',
             name: 'LIS_OLD_ARCHIVE_LINK',
             defaultValue: "${LIS_OLD_ARCHIVE_VERSION}",
-            description: 'Link to the previous lis version']
+            description: 'Link to the previous lis version'],
+        [$class: 'ChoiceParameterDefinition', choices: 'no\nyes',
+            name: 'FULL_RUN',
```
Comment (ader1990) [inline linux_pipeline/Jenkinsfile_lis_rpm_hyperv]: this param is confusing as to what means full run or no full run
And the name should be FULL_PERF_RUN

File: scripts/lis_hyperv_platform/trigger_perf_tests.ps1 [hunk 4]
```diff
@@ -1,25 +1,40 @@
...
-Import-Module "$scriptPath\ini.psm1"
+function lisPerf {
+    Write-Host "Triggering performance jobs for LIS"
+    Invoke-RestMethod -Method Post `
+        -Uri "${LocalJenkinsPerfURL}/buildWithParameters?LIS_URL=${LisUrl}&AZURE_TOKEN=${AzureToken}&`
+        FULL_RUN=${FullRun}&token=${LocalJenkinsPerfToken}"
+}
 
-function Main {
+function msftPerf {
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/trigger_perf_tests.ps1]: Keel the Verb-Noun naming
Trigger-MsftPerfRun

File: scripts/lis_hyperv_platform/trigger_perf_tests.ps1 [hunk 5]
```diff
@@ -1,25 +1,40 @@
...
+    [Parameter(ParameterSetName='msft',Mandatory=$false)][string]$KernelVersionPath,
+    [parameter(Mandatory=$true)][String] $LocalJenkinsPerfURL,
+    [parameter(Mandatory=$true)][String] $LocalJenkinsPerfToken
 )
 
 $ErrorActionPreference = "Stop"
 
-$scriptPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
-Import-Module "$scriptPath\ini.psm1"
+function lisPerf {
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/trigger_perf_tests.ps1]: Keel the Verb-Noun naming
Trigger-LisPerfRun

File: scripts/lis_hyperv_platform/trigger_perf_tests.ps1 [hunk 6]
```diff
@@ -1,25 +1,40 @@
...
-    Write-Host "Triggering performance jobs for kernel folder: ${kernelPath}"
+    -Section "KERNEL_BUILT" -Key "folder"
     Invoke-RestMethod -Method Post `
         -Uri "${LocalJenkinsPerfURL}/buildWithParameters?KERNEL=${kernelPath}&token=${LocalJenkinsPerfToken}"
 }
 
+function Main {
+    if ($LisUrl) {
+        lisPerf
+    }
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/trigger_perf_tests.ps1]: no need for extra line

**Pull request merged**